packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -122,6 +123,15 @@ export const useCheckoutPage = props => {
         }
     }, [cartId, getCheckoutDetails, getCheckoutStep, setCheckoutStep]);
 
+    const windowSize = useWindowSize();
```

Comment:
It feels odd to me to use this hook inside a talon since window size is presentational.

Reply:
Is there any other solution for this?

Reply:
What do you mean?

Reply:
Ok I moved this to the component. It did not feel like logic that belonged to a talon. We can discuss in general.